### ISSUE TITLE
installing rainbond Failed. how can I do.

### ISSUE THREAD

**Issue by EvanCreno** [goodrain/rainbond-install#84]
```
Start install docker ...
manage01:
  Name: /opt/rainbond/envs/docker.sh - Function: file.managed - Result: Changed Started: - 21:12:56.495603 Duration: 62.932 ms
  Name: /etc/goodrain/envs/docker.sh - Function: file.managed - Result: Changed Started: - 21:12:56.558951 Duration: 44.633 ms
  Name: /etc/docker/daemon.json - Function: file.managed - Result: Changed Started: - 21:12:56.603929 Duration: 39.576 ms
  Name: docker-repo - Function: pkgrepo.managed - Result: Clean Started: - 21:12:56.752826 Duration: 738.719 ms
----------
      ID: gr-docker-engine
      Function: pkg.installed
      Result: False
      Comment: Error occurred installing package(s). Additional info follows:

              errors:
                  - Running scope as unit run-20072.scope.
                    Loaded plugins: fastestmirror, priorities
                    Loading mirror speeds from cached hostfile
                     * base: mirrors.aliyun.com
                     * extras: mirrors.aliyun.com
                     * updates: mirrors.aliyun.com
                    Resolving Dependencies
                    --> Running transaction check
                    ---> Package gr-docker-engine.x86_64 0:1.12.6-111.g1526e3f.el7.centos will be installed
                    --> Processing Dependency: gr-docker-engine-selinux >= 1.12.6-111.g1526e3f.el7.centos for package: gr-docker-engine-1.12.6-111.g1526e3f.el7.centos.x86_64
                    --> Processing Dependency: libltdl.so.7()(64bit) for package: gr-docker-engine-1.12.6-111.g1526e3f.el7.centos.x86_64
                    --> Running transaction check
                    ---> Package gr-docker-engine-selinux.noarch 0:1.12.6-111.g1526e3f.el7.centos will be installed
                    ---> Package libtool-ltdl.x86_64 0:2.4.2-22.el7_3 will be installed
                    --> Processing Conflict: gr-docker-engine-selinux-1.12.6-111.g1526e3f.el7.centos.noarch conflicts docker-selinux
                    --> Processing Conflict: gr-docker-engine-selinux-1.12.6-111.g1526e3f.el7.centos.noarch conflicts docker-engine-selinux
                    --> Finished Dependency Resolution
                    Error: gr-docker-engine-selinux conflicts with 2:container-selinux-2.55-1.el7.noarch
                     You could try using --skip-broken to work around the problem
                     You could try running: rpm -Va --nofiles --nodigest
     Started: 21:12:57.506377
     Duration: 9043.717 ms
     Changes:

Summary for manage01
------------
Succeeded: 4 (changed=3)
Failed:    1
------------
```

**Comment by ysicing** [comment]
```
yum install gr-docker-engine -y 
```
Can you tell me the result?

**Comment by EvanCreno** [comment]
@ysicing 

master branch, 
I used 
> yum install gr-docker-engine -y
It doesn't seem to work.
 
```
Start install docker ...
manage01:
  Name: /opt/rainbond/envs/docker.sh - Function: file.managed - Result: Clean Started: - 21:24:27.457356 Duration: 53.008 ms
  Name: /etc/goodrain/envs/docker.sh - Function: file.managed - Result: Clean Started: - 21:24:27.510689 Duration: 28.991 ms
  Name: /etc/docker/daemon.json - Function: file.managed - Result: Clean Started: - 21:24:27.539950 Duration: 23.055 ms
  Name: docker-repo - Function: pkgrepo.managed - Result: Clean Started: - 21:24:27.670887 Duration: 658.08 ms
----------
    ID: gr-docker-engine
    Function: pkg.installed
    Result: False
    Comment: Error occurred installing package(s). Additional info follows:

              errors:
                  - Running scope as unit run-26977.scope.
                    Loaded plugins: fastestmirror, priorities
                    Loading mirror speeds from cached hostfile
                     * base: mirrors.aliyun.com
                     * extras: mirrors.aliyun.com
                     * updates: mirrors.aliyun.com
                    Resolving Dependencies
                    --> Running transaction check
                    ---> Package gr-docker-engine.x86_64 0:1.12.6-111.g1526e3f.el7.centos will be installed
                    --> Processing Dependency: gr-docker-engine-selinux >= 1.12.6-111.g1526e3f.el7.centos for package: gr-docker-engine-1.12.6-111.g1526e3f.el7.centos.x86_64
                    --> Processing Dependency: libltdl.so.7()(64bit) for package: gr-docker-engine-1.12.6-111.g1526e3f.el7.centos.x86_64
                    --> Running transaction check
                    ---> Package gr-docker-engine-selinux.noarch 0:1.12.6-111.g1526e3f.el7.centos will be installed
                    ---> Package libtool-ltdl.x86_64 0:2.4.2-22.el7_3 will be installed
                    --> Processing Conflict: gr-docker-engine-selinux-1.12.6-111.g1526e3f.el7.centos.noarch conflicts docker-selinux
                    --> Processing Conflict: gr-docker-engine-selinux-1.12.6-111.g1526e3f.el7.centos.noarch conflicts docker-engine-selinux
                    --> Finished Dependency Resolution
                    Error: gr-docker-engine-selinux conflicts with 2:container-selinux-2.55-1.el7.noarch
                     You could try using --skip-broken to work around the problem
                     You could try running: rpm -Va --nofiles --nodigest
     Started: 21:24:28.340269
     Duration: 8333.84 ms
     Changes:

Summary for manage01
------------
Succeeded: 4
Failed:    1
------------
Total states run:     5
Total run time:   9.097 s
ERROR: Minions returned with non-zero exit code
```

**Comment by ysicing** [comment]
`gr-docker-engine-selinux` conflicts with `container-selinux-2.55-1.el7.noarch`

just run:`yum remove container-selinux`

**Comment by EvanCreno** [comment]
It's worked. 3Q.

**Comment by ysicing** [comment]
@EvanCreno ok

**Comment by ysicing** [comment]
fixed [#6cb8d88](https://github.com/goodrain/rainbond-install/commit/6cb8d889655ab56c013e9f9ed363ecf64f03b5c0)
